myia/compile/backends/relay_helpers.py

```diff
@@ -180,6 +181,12 @@ def to_relay_type(self, a: AbstractScalar):
         return relay.ty.scalar_type(type_to_np_dtype(tp))
 
 
+@overload  # noqa: F811
+def to_relay_type(self, a: AbstractType):
+    # Let's just replace abstract type object by type wrapped inside it.
```

Comment:
That's a bit fishy. As far as I understand, values with type AbstractType are not really used, so I think we can just flat out replace it with an int type.

Reply:
OK!